samples/wifi/shell/src/wifi_shell.c

```diff
@@ -0,0 +1,341 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/* @file
+ * @brief WPA supplicant shell module
```

Comment:
```suggestion
 * @brief Wi-Fi shell module
```
This line looks outdated